en/08_Changelogs/6.0.0.md

```diff
@@ -898,6 +898,14 @@ The following classes have been changed when code was moved to the `silverstripe
 |`SilverStripe\ExternalLinks\Tasks\CurlLinkChecker`|[`SilverStripe\Reports\ExternalLinks\Tasks\CurlLinkChecker`](api:SilverStripe\Reports\ExternalLinks\Tasks\CurlLinkChecker)|
 |`SilverStripe\ExternalLinks\Tasks\LinkChecker`|[`SilverStripe\Reports\ExternalLinks\Tasks\LinkChecker`](api:SilverStripe\Reports\ExternalLinks\Tasks\LinkChecker)|
 
+The following subclasses of [`Validator`](api:SilverStripe\Forms\Validator) have been renamed:
```

Comment:
That's a link to the old FQCN and won't work here.

Reply:
Updated